static/js/apps/eval_retrieval_generation/app.tsx

```diff
@@ -93,6 +93,14 @@ export function App(props: AppPropType): JSX.Element {
     }
   }
 
+  // Sign in automatically.
+  useEffect(() => {
+    const buttons = Array.from(document.getElementsByTagName("button"));
+    const signInButton = buttons.find((el) => el.innerText.includes("Sign In"));
```

Comment:
nit: wonder if there is a better way to find this button: maybe through className or id?

& same thing for sxs app

Reply:
I'll do you one better and bypass the button entirely!